examples/basics/visuals/text_visual.py

```diff
@@ -1,23 +1,77 @@
-# -*- coding: utf-8 -*-
-# vispy: gallery 30
-# -----------------------------------------------------------------------------
-# Copyright (c) Vispy Development Team. All Rights Reserved.
-# Distributed under the (new) BSD License. See LICENSE.txt for more info.
-# -----------------------------------------------------------------------------
-
 from vispy import app, gloo, visuals
+import numpy as np
 
 
 class Canvas(app.Canvas):
     def __init__(self):
         app.Canvas.__init__(self, title='Glyphs', keys='interactive')
+
         self.font_size = 48.
+
+        # Create a cross eye of lines for easy see if text anchor positions are good
+        l_pos = np.array([[-1.0,0.0], [1.0,0.0], [0.0,0.0], [0.0,1.0], [0.0,-1.0]])
+        self.cross_eye_line = visuals.LineVisual(pos=l_pos, color=(1.0, 0.0, 0.0, 1), method='gl')
+
+        """
+        Standard C escape sequences:
+            -   \a  ->  alert (beep, bell)
+            -   \b  ->  Backspace
+            -   \f  ->  Formfeed
+            -   \n  ->  Newline
+            -   \r  ->  Carriage Return
+            -   \t  ->  Horizontal Tab
+            -   \v  ->  Vertical Tab
+            -   \\  ->  Backslash
+            -   \'  ->  Single quotation mark
+            -   \"  ->  Double quotation mark
+        """
+        the_big_test_string = 'This is the big test string!\n'
+        the_big_test_string += 'It includes all of the esqape sequences known\n'
```

Comment:
`escape` typo.

Additionally, the font size of 48 is really large on my laptop screen. I had to make it 12 to even see a majority of the text. This could be a HiDPI display issue too.

Reply:
Ah I see scrolling changes the size. Maybe start with a smaller font so users can actually see that message ;).

And if I'm being picky, could you put the list of escape sequences in the docstring of the canvas class or the canvas init method?

Reply:
I agree. Your pickiness is very much welcome! ;)

Reply:
@djhoese Better?